content/docs/setup/kubernetes/additional-setup/mesh-expansion/index.md

```diff
@@ -266,12 +266,15 @@ The `server: envoy` header indicates that the sidecar intercepted the traffic.
     {{< text bash >}}
     $ istioctl  register -n ${SERVICE_NAMESPACE} vmhttp ${GCE_IP} 8080
     {{< /text >}}
+    
+    Make sure you have already added `istioctl` client to your PATH environment variable, as described
+    in step 3 in the [Download page](/docs/setup/kubernetes/download/#download-and-prepare-for-the-installation).
```

Comment:
Should we move this up before `istioctl` command?

Reply:
Done. Also put PATH in backticks in the Download page.